tests/console.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from '@playwright/test';
+import { loadPage } from './libs/load-page';
+import pkg from './../packages/app/package.json';
+
+loadPage();
+
+test.describe('web console', () => {
+  test('editor version', async ({ page }) => {
+    // https://playwright.dev/docs/evaluating
+    // https://github.com/microsoft/playwright/issues/13059
+    // Get the handle to a specific function.
+    //Later on, call this function with some arguments.
+    // const msg = await getEditoVersionHandle.evaluate((post, args) => post);
+    // console.log(getEditoVersionHandle);
+    await page.waitForTimeout(500);
```

Comment:
I think this wait should be placed on the `loadPage`.

Reply:
> I think this wait should be placed on the `loadPage`.

I remove the   `page.waitForTimeout(1000)` in all scene  before test,  it is a litte redundant.  it is ok now , waitForTimeout is not necessary  for  every test cases